tt_llk_blackhole/common/inc/sfpu/ckernel_sfpu_binary_bitwise.h

```diff
@@ -20,16 +20,19 @@ enum class BinaryBitwiseOp : uint8_t
     XOR = 2,
 };
 
-template <bool APPROXIMATION_MODE, BinaryBitwiseOp BITWISE_OP, int ITERATIONS = 8>
+using DataType = InstrModLoadStore;
+
+template <bool APPROXIMATION_MODE, BinaryBitwiseOp BITWISE_OP, DataType DTYPE = INT32, int ITERATIONS = 8>
```

Comment:
```suggestion
template <bool APPROXIMATION_MODE, BinaryBitwiseOp BITWISE_OP, InstrModLoadStore INSTRUCTION_MODE = INT32, int ITERATIONS = 8>
```
And then we don’t use DataType since it sounds a bit misleading.